install/reproducibility.qmd

```diff
@@ -54,3 +54,134 @@ prefix <- ifelse (.Platform$OS.type == "windows", "file:///", "file://")
 repos <- paste0(prefix, normalizePath(snapshot, "/"))
 install.packages(c("V8", "mongolite"), repos = repos)
 ```
+
+### Using the S3 API {#s3}
+
+R-universe also exposes a partial [S3-compatible API](https://docs.aws.amazon.com/AmazonS3/latest/API/Type_API_Reference.html) that you can use to list, download, or mirror package files. 
+
+In R, you can use the [{paws}](https://paws-r-sdk.github.io/) package to access the S3 API. Note that it requires setting the
+`s3_virtual_address` configuration option to `TRUE` and anonymous credentials:
+
+```r
+library(paws)
+client <- paws::s3(
+  config = list(
+    endpoint = "https://r-universe.dev",
+    s3_virtual_address = TRUE
+  ),
+  credentials = list(anonymous = TRUE),
+  region = "any_value_here_works"
```

Comment:
even this value or an actual region code? Maybe add a region code and "any value here works" as code comment instead?